src/CattyTests/DataModel/VariablesContainerTest.swift

```diff
@@ -236,32 +233,32 @@ final class VariablesContainerTest: XCTestCase {
         container.setUserVariable(userVariable1, toValue: 10)
         container.change(userVariable1, byValue: 10)
 
-        var variables = container.objectVariables(for: objectA) as! [UserVariable]
+        var variables = container.objectVariables(for: objectA)
 
-        XCTAssertEqual(1, variables.count)
-        XCTAssertEqual(variables[0].value as! Int, 20)
+        XCTAssertEqual(1, variables?.count)
+        XCTAssertEqual(variables?[0].value as! Int, 20)
 
         container.change(userVariable1, byValue: 10)
 
-        variables = container.objectVariables(for: objectA) as! [UserVariable]
+        variables = container.objectVariables(for: objectA)
 
-        XCTAssertEqual(1, variables.count)
-        XCTAssertEqual(variables[0].value as! Int, 30)
+        XCTAssertEqual(1, variables?.count)
+        XCTAssertEqual(variables?[0].value as! Int, 30)
     }
 
     func testAddToUserList() {
         let objectA = SpriteObject()
         objectA.name = "testObjectA"
 
-        let list1 = UserVariable(name: "testVariable1", isList: true)
+        let list1 = UserList(name: "testName")
 
         let container = VariablesContainer()
 
         container.addObjectList(list1, for: objectA)
-        container.add(toUserList: list1, value: 10)
-        container.add(toUserList: list1, value: 20)
+        container.add(to: list1, value: 10)
+        container.add(to: list1, value: 20)
 
-        let lists = container.allLists() as! [UserVariable]
+        let lists = container.allLists() as! [UserList]
```

Comment:
Please remove the force cast once `NSMutableArray` has been changed to `NSMutableArray` in the `VariablesContainer`.